workflow/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+# This script is used by CI to build for a specific flavor.  It can be used
+# locally: `./workspace/build.sh [linux|android]`
```

Comment:
When running `./workspace/build.sh android` directly, it seems like it may be necessary to inform the script where to find the NDK as it currently hard-codes `/opt/android-ndk`.